CHANGELOG.md

```diff
@@ -14,8 +14,15 @@ title: Changelog
 
 * `<cc-doc-card>`: rename title property to heading to fix a conflict.
 * `<cc-doc-list>`: use heading when initializing the cards instead of title.
-* `<cc-input-text>`: change default font-family to inherit instead of monospace, add CSS Custom Prop to change the `<input>` font-family.
-* `<cc-input-number>`: change default font-family to inherit instead of monospace, add CSS Custom Prop to change the `<input>` font-family.
+* `<cc-input-text>`: change default font-family to `inherit` instead of `monospace`. Add CSS Custom Prop to change the `<input>` font-family.
+* `<cc-input-number>`: change default font-family to `inherit` instead of `monospace`. Add CSS Custom Prop to change the `<input>` font-family.
+* `<cc-pricing-estimation>`: change default background of the recap to `--cc-color-bg-primary`. Add CSS Custom Prop to change the recap background.
+* `default-theme` and most of the components:
+  * move `default-theme` (design tokens) from a JavaScript file to a CSS file.
+  * remove `default-theme` import from all components.
+  * prefix decision design tokens with `cc-`.
+  * define default text color using `cc-color-text-default` where necessary.
```

Comment:
Most of the time, I chose not to define the default text color inside a component because I think it should simply inherit it from the context (much like native HTML elements do).

There are only a few exceptions (shoelace component) where I defined a default text color because inheritance could not work for these.